src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -699,6 +704,22 @@ public ResponseEntity<List<MeasurementVariable>> getStudyVariablesByVariableType
 		return new ResponseEntity<>(variables, HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Validate Alias input for a Dataset Variable", notes = "Validate Alias input for a Dataset Variable")
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES', 'MANAGE_STUDIES')")
+	@RequestMapping(value = "/{crop}/programs/{programUUID}/variable/{variableId}/validate-alias", method = RequestMethod.POST)
+	public ResponseEntity<Boolean> validateVariableAlias(
```

Comment:
@cheryleigh Instead of creating a new endpoint for validation of alias. I realized, we can just use the existing search variable api in VariableResource: /{cropName}/programs/{programUUID}/variables/search to search for a variable by name or alias.